downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -11,6 +11,8 @@ Use this example to populate the inventory file to deploy single instances of {C
 
 * {EDAController} must be installed on a separate server and cannot be installed on the same host as {HubName} and {ControllerName}.
 
+* {EDAController} cannot be installed in a high availability or clustered configuration. Ensure there is only one host entry in the `automationedacontroller` section of the inventory.
```

Comment:
@g-murray - thanks for the peer review of sentence added here. I've taken a look at this and I feel that this is an exception. The scope of this jira ticket is to only add the point that EDA cannot be installed in an HA or clustered environment, which is the sentence in green. If you don't mind, could you merge the portion I've added to that section? We can create a separate ticket to attempt to make all of the bullets parallel, if possible. The last bullet in the Important section (there are 4) is completely different than the rest. So it might not be 100% possible in this case.